Makefile

```diff
@@ -1,11 +1,12 @@
 DEPS = 
-LLVM_VERSION = 8
+#LLVM_VERSION ="-8"
```

Comment:
When you install a specific version of LLVM in Ubuntu, the default `clang` `clang++` are not available. So let's keep the `LLVM_VERSION` variable as-is.